client/src/components/scores/scores.js

```diff
@@ -3,23 +3,37 @@
 export default {
 	template: `
 	<div class="top-bar secondary">
-	    <div class="top-bar-left">
-	        <input name="search" type="text" ng-model="scores.search" placeholder="search Scores" />
-	        <button class="button" ng-hide="scores.duplicateScores().length === 0" ng-click="scores.showDuplicates = !scores.showDuplicates">
-	            {{ scores.showDuplicates ? 'Show all scores' : 'Show only duplicates' }}
-	        </button>
-	        <button class="button" ng-hide="scores.errorScores().length === 0"
-	        ng-click="scores.showErrors = !scores.showErrors">
-	        {{ scores.showErrors ? 'Show all scores' : 'Show only errors'}}
-	        </button>
-	    </div>
-	</div>
-	<div class="top-bar-page">
-	    <div id="scores-list" class="grid-x grid-padding-x small-up-1 medium-up-3 large-up-5">
-	        <div ng-repeat="score in scores.scores()" class="cell grid-y">

```

Comment:
Why did you remove this class? It was meant to set equals height to the score cards

Reply:
I did not remove it if I am understanding properly what class you are talking about (the top-bar-left of the div). Perhaps it shows it like this because of indentation/because I used the menu class here and changed a lot of the things inside?

Reply:
Look at the diff without spaces (In the "changes" page, under "diff settings" enable "hide whitespace changes"). Then you can see what I mean. the `grid-y` class has been removed from the `cell`.